packages/@uppy/xhr-upload/src/index.js

```diff
@@ -190,6 +195,11 @@ module.exports = class XHRUpload extends Plugin {
   }
 
   createBareUpload (file, opts) {
+    // Set `file.data.type` in the blob to `file.meta.type`,
+    // because we might have detected a more accurate file type in Uppy
+    // https://stackoverflow.com/a/50875615
+    file.data = file.data.slice(0, file.data.size, file.meta.type)
```

Comment:
This doesn't have any effect for bare uploads, those only send the raw bytes

Reply:
Thanks! Removed from here.